microshift_release_notes/microshift-4-13-release-notes.adoc

```diff
@@ -80,8 +80,18 @@ For more information, see xref:../microshift_support/microshift-etcd.html#micros
 //
 // This content will be added post-GA, as it is asynchronous content.
 
-//[id="microshift-4-13-networking"]
-//=== Networking
+[id="microshift-4-13-networking"]
+=== Networking
+
+[id="microshift-4-13-load-balancer"]
+==== Deploying network load balancers on {product-title}
+
+{product-title} now offers a built-in implementation of network load balancers. For more details, see xref:../microshift_networking/microshift-networking.adoc#microshift-deploying-a-load-balancer_microshift-networking[Deploying a TCP load balancer on a workload].
+
+[id="microshift-4-13-blocking-nodeport"]
+==== Blocking a NodePort service on a host interface 
+
+You can now block a NodePort service on a specific host interface and restrict access from outside a {product-title} node. For more information, see  xref:../microshift_networking/microshift-networking.html#microshift-blocking-nodeport-access_microshift-networking[Blocking external access to the NodePort service on a specific host interface].
```

Comment:
This isn't really a new feature. The procedure describes how to use existing tools outside of MicroShift (nft) to accomplish the goal.

Reply:
Yeah thats a great point, its not really needed. I'll remove this note from the section. Thank you!